src/test/scala/org/camunda/dmn/DmnEngineConfigurationTest.scala

```diff
@@ -69,6 +71,16 @@ class DmnEngineConfigurationTest extends AnyFlatSpec with Matchers {
     result.map(_.value should be("Hello DMN"))
   }
 
+  "A DMN engine with escaped names" should "evaluate a decision with spaces in item definitions and item components" in {
```

Comment:
:wrench: You can use the `it` keyword here, instead of defining the `A DMN engine with escaped names` subject all over again. It's just a best practice that we like to observe.

In `ScalaTest` you only need to define the `subject` in the "test sentence" in the first test. Any consecutive tests just need to use the `it` keyword, and the same `subject` will be used. You can read more about it [here](https://www.scalatest.org/user_guide/writing_your_first_test).

```suggestion
  it should "evaluate a decision with spaces in item definitions and item components" in {
```